test_scripts/API/AddCommand/005_ATF_AddCommand_UI_AddCommand_WARNINGS.lua

```diff
@@ -0,0 +1,142 @@
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)   
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+local resultCodes = {"SUCCESS", "WARNINGS", "WRONG_LANGUAGE", "RETRY", "SAVED"}
+  for i=1,#resultCodes do
+    Test["TestStep_AddCommand_UI_AddCmd_WARNINGS_and_VR_AddCmd_"..resultCodes[i]] = function(self)
+    local cid = self.mobileSession:SendRPC("AddCommand",
+    {
+      cmdID = 11,
+      menuParams =
+      {
+        parentID = 0,
+        position = 0,
+        menuName ="Commandpositive"
+      },
+      vrCommands =
+      { 
+        "VRCommandonepositive",
+        "VRCommandonepositivedouble"
+      },
+      cmdIcon =
+      { 
+        value ="icon.png",
+        imageType ="DYNAMIC"
+      }
+    })
+    EXPECT_HMICALL("UI.AddCommand",
+    {
+      cmdID = 11,
+      cmdIcon =
+      {
+        value = storagePath .."icon.png",
+        imageType = "DYNAMIC"
+      },
+      menuParams =
+      {
+        parentID = 0,
+        position = 0,
+        menuName ="Commandpositive"
+      }
+    })
+    :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, "UI.AddCommand", "WARNINGS", {})
+    end)
+    
+    EXPECT_HMICALL("VR.AddCommand", 
+    {
+      cmdID = 11,
+      vrCommands =
+      {
+        "VRCommandonepositive",
+        "VRCommandonepositivedouble"
+      },
+      type = "Command"
+    })
+    :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, "VR.AddCommand", resultCodes[i], {})
+    end)
+    
+    EXPECT_RESPONSE(cid, { success = true, resultCode = "WARNINGS" })
+    EXPECT_NOTIFICATION("OnHashChange")
+  end
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+
+function Test.Postcondition_Restore_preloaded_file()
+  commonPreconditions:RestoreFile("sdl_preloaded_pt.json")
+end
+
+function Test.Postcondition_SDLStop()
+  StopSDL()
+end
+
+return Test
```

Comment:
@vvvakulenko, please after update use beautifier.sh and check if there are errors in script, using LuaChecker.